components/FeaturedJobItem/FeaturedJobItem.js

```diff
@@ -25,32 +24,34 @@ FeaturedJobItem.defaultProps = {
 
 function FeaturedJobItem({ title, source, sourceUrl, city, state, country, description, remote }) {
   return (
-    <article className={styles.job}>
+    <article className="py-4 px-0">
       <OutboundLink href={sourceUrl} analyticsEventLabel={`Featured Job ${source}`}>
         <h6>{title}</h6>
       </OutboundLink>
 
-      <div className={styles.details}>
-        <div className={styles.detailsContainer}>
-          <BuildingIcon className={styles.icon} />
-          <span className={styles.detail}>{source}</span>
+      <div className="flex flex-wrap text-lg text-secondary opacity-80 mt-1 ">
+        <div className="flex items-center gap-[0.3125rem]">
+          <BuildingIcon className="fill-secondary opacity-80 h-[0.825rem]" />
```

Comment:
let's use `h-3.5` instead of `h-[0.825]` in all instances on this file

and let's use `gap-1.5` instead of `gap-[0.3125rem]`

Reply:
done